firefox-ios/Client/metrics.yaml

```diff
@@ -656,6 +656,67 @@ downloads:
       - fx-ios-data-stewards@mozilla.com
     expires: "2025-01-01"
 
+# Microsurvey
+microsurvey:
+  shown:
+    type: event
+    description: |
+        The survey surface (bottom sheet) was shown and visible.
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/issues/19506
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/20801
+    notification_emails:
```

Comment:
wasn't sure if I should mark this with `data_sensitivity` since other places don't include it (but some do). I marked it as 2 in the data review request form though based on other request forms I saw. Let me know what the best way to proceed is, thanks!

Reply:
Looks fine to me.